setup.py

```diff
@@ -37,7 +37,7 @@ def is_platform_mac():
     return sys.platform == "darwin"
 
 
-min_cython_ver = "0.29.21"  # note: sync with pyproject.toml
+min_cython_ver = "0.29.24"  # note: sync with pyproject.toml
```

Comment:
do we have any docs that list the cython requirement and need updating?

Reply:
Just the whatsnew. I'll try to update this PR soonish.

Reply:
can we get away with not updating cython? if so then i think we can backport this

Reply:
Nope.